app/views/common/500.html.erb

```diff
@@ -1,21 +1,18 @@
 <%= alert :header => _("Oops, we're sorry but something went wrong"), :class => 'alert-danger',
   :actions => link_to(_('Back'), main_app.root_path, :class => 'btn btn-default'),
-  :text => h(exception.message),
+  :text => h(exception_message),
   :close => false %>
 
 <div class="pull-right">
   <%= documentation_button("7.3GettingHelp")%>
 </div>
+
 <p id="message">
   <%= _("If you feel this is an error with Foreman itself, please open a new issue with") %>
-  <%= link_to _("Foreman ticketing system"), "http://theforeman.org/projects/foreman/issues", :rel => "external" %>,
-  <%= _("You would probably need to attach the") %>
-  <%= link_to_function(_("Full trace"), "$('#backtrace').toggle()")%>
-  <%= _(", relevant log entries, and it is highly recommended to also attach the foreman-debug output.") %>
+  <%= link_to _("Foreman ticketing system"), "https://projects.theforeman.org/", :rel => "external" %>,
+  <%= _("Please include in your report the full error log that can be acquired by running: ") %>
+  <strong> foreman-rake errors:fetch_log request_id=<%= request_id %></strong>
+  <%= _(" and it is highly recommended to also attach the foreman-debug output.") %>
+  <%= _("The error code is ") %>
```

Comment:
Would it be easier for translations to use:
```suggestion
  <%= _("The error code is %s") % exception_code %>
```

Reply:
Please do not show error code, Foreman error codes (ERF) are different thing for different usecase. They will appear automatically in the exception message.

You want to call this code (or token) as "indident number" or "incident token" maybe "incident code".

I'd also add some information to hand over the number to the site administrator for further investigation.